src/OrchardCore.Modules/OrchardCore.Templates/Views/Template/Index.cshtml

```diff
@@ -2,6 +2,7 @@
 @{
     int startIndex = (Model.Pager.Page - 1) * (Model.Pager.PageSize) + 1;
     int endIndex = startIndex + Model.Templates.Count - 1;
+    var formAction = Model.Options.AdminTemplates ? "admin" : "index";
```

Comment:
Could we use `nameOf` if the options are strongly typed

Reply:
Is it necessary?

Reply:
If this is the only place it's fine